x-pack/plugins/security_solution/common/detection_engine/schemas/request/add_prepackged_rules_schema.test.ts

```diff
@@ -1596,5 +1596,84 @@ describe('add prepackaged rules schema', () => {
       };
       expect(message.schema).toEqual(expected);
     });
+
+    describe('threat_mapping', () => {
+      test('You can set a threat query, index, mapping, filters on a pre-packaged rule', () => {
+        const payload: AddPrepackagedRulesSchema = {
+          ...getAddPrepackagedRulesSchemaMock(),
+          threat_query: '*:*',
```

Comment:
Nit: It might be useful to make a `getThreatMatchAddOnMock` that includes these threat match values to throw in throughout.

Reply:
Added both a `getAddPrepackagedThreatRulesSchemaMock` and a `getAddPrepackagedThreatRulesSchemaDecodedMock` here as well as the other schema based tests to keep the boiler platting down as suggested. 👍 

So now it looks like all the rest. Thanks for the good advice